converged_sound_open_fw_modules_development/FW/intel_common/module_binmaps/aca_module.binmap

```diff
@@ -0,0 +1,35 @@
+module o ACA
+uuid 88881111-2222-3333-4444-555566667777
+name Aca module example
+version_major 0x1
+version_minor 0x0
+version_hotfix 0x0
+version_build 0x0
+affinity_mask MASTER_CORE_AFFINITY
+instance_count +1
+domain_types DP
+type AudClassModule
+stack_size 1000
```

Comment:
0x1000 ?

Reply:
Yes, this is a hex value. This is described in README.md file under ```Modules Configuration``` in #4222 .

Reply:
ok, great - but we have some values already using 0xNNNN so we should be consistent as this could confuse.